File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L5/05/pengulangan.ts
import { forEach } from 'lodash';

class inputoutput {
  static hitHasil(m: number): void {
    for (let a = m; a <= 10; a++) {
      console.log(`${a}\t\t${a * 1.609}`);
    }
  }

  public static main(args: string[]): void {
    const m = 1;
    console.log('Miles\t\tKilometers');
    console.log('-------------------------------');
    this.hitHasil(m);
  }
}

inputoutput.main(process.argv.slice(2));

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L4/05/pengulangan.kt
package IR

fun hitHasil(m: Int) {
    var m = m
    while (m <= 10) {
        println("$m\t\t${m * 1.609}")
        m++
    }
}

fun main(args: Array<String>) {
    var m = 1
    println("Miles\t\tKilometers")
    println("-------------------------------")
    hitHasil(m)
}

